src/QuickMarcEditor/QuickMarcCreateWrapper.js

```diff
@@ -102,6 +108,14 @@ const QuickMarcCreateWrapper = ({
   }, [onSave, marcType]);
 
   const onSubmit = useCallback(async (formValues) => {
+    const lccnDuplicationError = await validateLccnDuplication(formValues);
```

Comment:
Can we do this valdiation in `QuickMarcEditor` component? We can reduce duplication this way